src/sentry/static/sentry/app/views/organizationEventsV2/modalLineGraph.jsx

```diff
@@ -49,6 +59,55 @@ const getCurrentEventMarker = currentEvent => {
   };
 };
 
+/**
+ * Handle click events on line markers
+ *
+ * When a user clicks on a marker we want to update the modal
+ * to display an event from that time slice. While each graph slice
+ * could contain thousands of events, we use the /latest endpoint
+ * to pick one.
+ */
+const handleClick = async function(
+  series,
+  {api, organization, groupId, interval, selection, location}
+) {
+  // Get the timestamp that was clicked.
+  const value = series.value[0];
+
+  // Get events that match the clicked timestamp
+  // taking into account the group and current environment & query
+  const query = {
+    environment: selection.environments,
+    query: location.query.query,
+    group: groupId,
+    start: getUtcDateString(value),
+    end: getUtcDateString(value + intervalToMilliseconds(interval)),
+  };
+
+  const url = `/organizations/${organization.slug}/events/latest/`;
+  let response;
+  try {
+    response = await api.requestPromise(url, {
+      method: 'GET',
+      query,
+    });
+  } catch (e) {
+    // Do nothing, user could have clicked on a blank space.
```

Comment:
Thoughts (in general) about `console.error` in catch blocks? Sometimes they can end up swallowing errors in development and testing too.

Reply:
I think it makes sense when the failure is not expected to happen. I'm certain that users will click in blank spaces causing 404. I don't want to be notified in sentry when that happens though.

Reply:
`console.error` won't log to sentry, I'm mainly thinking about during dev/tests (e.g. in tests if you don't mock this endpoint, the test will pass, maybe not applicable here, but in other places it's more relevant)